lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -176,56 +176,56 @@ <h3 class="course-title">
                       ## Translators: The course name will be added to the end of this sentence.
                       % if not is_course_blocked:
                       <a href="#unenroll-modal" class="action action-unenroll" rel="leanModal" data-course-id="${course_overview.id | h}" data-course_overview-number="${course_overview.number | h}" data-dashboard-index="${dashboard_index}"
-                         onclick="set_unenroll_message('${_("Are you sure you want to unenroll from the purchased course %(course_number)s?")}', '${_("You will be refunded the amount you paid.")}')">
+                         onclick="set_unenroll_message('${_("Are you sure you want to unenroll from the purchased course %(course_name)s ($(course_number)s)?")}', '${_("You will be refunded the amount you paid.")}')">
```

Comment:
@sarina I'm sorry about that. I prepared a fix in https://github.com/edx/edx-platform/pull/10456